pkg/render/intrusion_detection.go

```diff
@@ -1504,20 +1518,24 @@ func (c *intrusionDetectionComponent) getBaseADDetectorsPodTemplate(podTemplateN
 						},
 						Env: []corev1.EnvVar{
 							{
-								Name: "ELASTIC_HOST",
+								Name: "ES_HOST",
```

Comment:
I think the preferred way would be to actually use the existing function `relasticsearch.ContainerDecorate(...)`. For an example, see compliance.go. We have a convenient function that normalizes all of these fields. Would it be possible to modify the adDetectorsImage, such that these standard fields can be continued to be supported?

Reply:
And this:
```
					{
						Name: "es-certs",
						VolumeSource: corev1.VolumeSource{
							Secret: &corev1.SecretVolumeSource{
								SecretName: relasticsearch.PublicCertSecret,
								Items: []corev1.KeyToPath{
									{Key: "tls.crt", Path: "es-ca.pem"},
								},
							},
						},
					},
```
Can then be removed.

Reply:
https://github.com/tigera/operator/pull/2074/commits/72b90570405d14a3140de0aabb467ac8db22db4a - set to ELASTIC_* in AD as well